application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -146,6 +132,40 @@ private boolean isUserOnCooldown(User user) {
             .isPresent();
     }
 
+    private void sendCooldownResponse(SlashCommandInteractionEvent event) {
+        User user = event.getUser();
+
+        String message =
+                """
+                        Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread %s instead.
+                        If you made a typo or similar, you can adjust the title using the command %s and the category with %s :ok_hand:""";
```

Comment:
put the smiley as plain utf-8 symbol in there, like, put 👌 in there. java source code supports utf-8

Reply:
done